crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -1499,13 +1522,13 @@ async fn handle_call<G: Fn(Vec<Effect>) + Send + Sync>(
                             errors::failed_to_analyse::ecmascript::NODE_PRE_GYP_FIND.to_string(),
                         ),
                     );
-                    return Ok(());
```

Comment:
early return is definitely nicer to read than an else branch (kinda applies to a lot of the changes here)